front/lib/resources/run_resource.ts

```diff
@@ -108,7 +109,19 @@ export class RunResource extends BaseResource<RunModel> {
   }
 
   static async deleteAllByAppId(appId: ModelId, transaction?: Transaction) {
-    // TODO: First, delete the run_usage before deleting the run.
+    assert(typeof appId === "number");
+    await RunUsageModel.destroy({
```

Comment:
Do you know why they were not deleted before? 
Asking because of this sentence "Runs are only deleted in core but will remain in front since we have bigger plan for them." from https://github.com/dust-tt/dust/pull/5642

Reply:
My understanding is that we delete runs in core periodically but keep them in front. But in case of a GDPR delete I think we have not choice but to delete everything?

We could potentially try to keep RunUsage objects around but they would really be left dangling...